includes/gateways/stripe/includes/admin/admin-actions.php

```diff
@@ -388,3 +456,96 @@ function give_stripe_show_currency_notice() {
 }
 
 add_action( 'admin_notices', 'give_stripe_show_currency_notice' );
+
+/**
+ * Disconnect Stripe Account.
+ *
+ * @since 2.7.0
+ *
+ * @return void
+ */
+function give_stripe_disconnect_connect_stripe_account() {
+	$get_data = give_clean( $_GET );
+
+	if ( is_admin() && isset( $get_data['stripe_disconnected'] ) ) {
```

Comment:
@mehul0810 no need to check `is_admin` on `admin_init` hook and please check for user capabilities + nonce other any other user can delete stripe account but just visiting URL. `manual` type connection is easy to find.